.circleci/config.yml

```diff
@@ -32,17 +32,17 @@ references:
     resource_class: large
     working_directory: ~/aeternity
     environment:
-      OTP_VERSION: 20.3.8.24
-  container_otp_plt: &container_otp_plt _build/default/rebar3_20.3.8.24_plt
+      OTP_VERSION: 22.3.4.10
```

Comment:
It seems that erlang repository does not have a package with this version: https://packages.erlang-solutions.com/erlang/debian/pool/

The closest/latest I see is `22.3.4.9`